components/tab/TitleTab.js

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import { Heading4 } from '../typography';
+
+class TitleTab extends PureComponent {
```

Comment:
Why is it called `TitleTab` and not just `Tab`. Or do you still have to create the IconTab? I want to see a story with icons as well.

Reply:
Indeed, an IconTab is on its way, but not yet.

 :ok: by @InstaK on 29/11/2017 16:35:49

Reply:
Can you add an `onClick` to the component (and on IconTab as well). If you now click on an item, it keeps the focus state, which is default behaviour, but kind of annoying and ugly. So you can trigger a blur in the onClick of the element. I did the same in the `Checkbox` component, you can use that as an example.